scripts/linters/js_ts_linter.py

```diff
@@ -1117,6 +1117,41 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from eslint messages.
+
+        Args:
+            lint_messages: str. Messages returned by the js_ts linter.
+
+        Returns:
+            str. A string with the trimmed messages.
+        """
+        error_messages = ''
+        # Extract the message from list and split the message by newline
+        # so that we can use them and remove last four lines from the end.
+        # Becuase last two lines are empty with a newline character and
+        # third one have a message with number of errors.
+        # 1 errors found in 1 file
+        # and fourth one with a message to fix automatically.
```

Comment:
Btw does all this stuff hold in the case of successful linting? Ditto for the other classes above too.